homeassistant/helpers/entity_registry.py

```diff
@@ -924,13 +924,22 @@ async def async_migrate_entries(
 
 
 @callback
-def async_resolve_entity_ids(
+def async_validate_entity_ids(
```

Comment:
This could be considered a breaking change for custom integrations.

Reply:
I checked that before I merged this PR. I have a folder containing all custom integrations from HACS and could not find any usage.

Reply:
Ok.